src/data/faq.json

```diff
@@ -566,11 +566,14 @@
                         "anchor": "encounter_criteria",
                         "active": true,
                         "textblock": [
-                            "The Corona Warn App has three different status indicators: \"unknown risk\" (grey), \"low risk\" (green) and \"increased risk\" (red).",
+                            "The calculation of your individual risk is based on the following factors:",
+                            "<ol><li> duration of the encounter </li><li> Distance at the time of the encounter </li><li> Time that has elapsed since the encounter </li><li>Infectiousness (derived from information on when and whether symptoms occur).</li></ol>",
+                            "If the calculated risk exceeds a certain threshold specified by the Robert Koch Institute, users will receive a notification regarding an increased risk.",
```

Comment:
The German text is linked to https://github.com/corona-warn-app/cwa-documentation/blob/master/solution_architecture.md#risk-score-calculation so the English text should be linked also.

` "If the calculated risk exceeds a certain threshold specified by the Robert Koch Institute, users will receive a notification regarding an increased risk.",` =>

` "If the calculated risk exceeds a certain <a href='https://github.com/corona-warn-app/cwa-documentation/blob/master/solution_architecture.md#risk-score-calculation' target='_blank' rel='noopener noreferrer'>threshold</a> specified by the Robert Koch Institute, users will receive a notification regarding an increased risk.",`

Reply:
Thanks again !